test/e2e/singlecluster/e2e_test.go

```diff
@@ -118,6 +119,78 @@ var _ = ginkgo.Describe("Kueue", func() {
 			util.ExpectAllPodsInNamespaceDeleted(ctx, k8sClient, ns)
 		})
 
+		ginkgo.It("Should allow to schedule Jobs via CronJob", func() {
```

Comment:
Great test case! Thanks for adding it.

Reply:
Thanks, the only downside is that sometimes the Job gets 1min to be created by the CronJob, because CronJob seems to only create with 1min resolution. I'm wondering if we could trick it: 
- create CronJob suspended
- patch CronJob status to indicate the last schedule was over 1min ago
- resume

I will check if this works, because I don't like wasting 1min (or 30s on average) for this test.